src/main/resources/application.yml

```diff
@@ -134,3 +134,8 @@ notify:
 
 loadshedding:
   modulus: 10
+
+surveys:
+  - social
+  - asteroid
```

Comment:
I didn't expect this bit of the JIRA ticket to be taken quite so literally!
Could we lose asteroid and boat please, even those this will be overridden at deployment time, still uneasy about hokey config being included.

Reply:
Done